tests/io/test_structure.py

```diff
@@ -1397,3 +1397,48 @@ def test_given_invalid_values_raises_expectation(
             with pytest.raises(ValueError) as exc_err:
                 Dambreak.check_location(invalid_values)
             assert str(exc_err.value) == expected_err
+
+
+class TestOrifice:
+    """
+    Wrapper class to test all the methods and sublcasses in:
+    hydrolib.core.io.structure.models.py Orifice class.
+    """
+
+    def test_create_orifice(self):
+        structure = Orifice(**self._create_orifice_values())
```

Comment:
Would it not be nicer to do something like this (to avoid the code duplication for all values):
```
orifdict = **self._create_orifice_values()
structure = Orifice(orifdict)
for key, value in orifdict.items():
    assert get_attr(structure, key) == value

Reply:
As agreed, will not do this. I think it is easier to analyze when the test fails with the current solution.